lib/Pusher.php

```diff
@@ -330,6 +352,7 @@ public function trigger( $channels, $event, $data, $socket_id = null, $debug = f
 
 		if ( $socket_id !== null )
 		{
+			$this->validate_socket_id( $socket_id );
```

Comment:
I agree, but the reason I put it here was because if $socket_id is null, then there is no need to validate it because it will not be used. I guess I could treat null socket_ids as being valid in validate_socket_id(), and then I could add the call near the top of the function without needing to wrap it in another if statement.